src/core/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,63 @@
+import { Icon, IconType } from '../icon';
+import { Link, type ILinkProps } from '../link';
+import { Tag, type ITagProps } from '../tag';
+
+/**
+ * Object representing a sequential segment in the Breadcrumbs path.
+ */
+interface Breadcrumb extends ILinkProps {
+    label: string;
+}
+
+/**
+ * List sizing of Breadcrumb objects with a min length of 1 and max length of 3.
+ */
+export type UpToThreeCrumbs = Breadcrumb[] & { length: 1 | 2 | 3 };
+
+export interface IBreadcrumbsProps {
+    /**
+     * Array of three Breadcrumb objects {href: string, label: string}
+     * The array indicates depth from the current position to be displayed in the Breadcrumbs.
+     * Starting at index 0 you must define the root, up to the current location as desired.
+     */
+    breadcrumbOrder: UpToThreeCrumbs;
+    /**
+     * Text to be displayed at the end of the Breadcrumbs indicating current location in the sitemap.
+     */
+    currentPage: string;
```

Comment:
What if, instead, we only have one `links` or `items` array and render the last one as the "active" one (without the link but as a simple text)? Chakra-ui still render the active item through the `BreadcrumbItem` component (https://v2.chakra-ui.com/docs/components/breadcrumb/usage), same for Bootstrap (https://getbootstrap.com/docs/4.0/components/breadcrumb/)